middleware/publishable.js

```diff
@@ -34,6 +35,9 @@ module.exports = function *publishable(next) {
     return;
   }
 
+  // See more at npm/npm#9616 and cnpmjs/cnpmjs.org#1024
+  common.decodeScopedPackageName(this.params);
```

Comment:
This is, even worst.

Because you just decode the original name and pass the test, system will create a new package named `@kcauchy%2fpkg`, instead of `@kcauchy/pkg`, the decoded one, and because of its special name, you can't even be able to delete it or access it.

So, the only correct solution is just decode the parameters themselves.